app/views/TotalNumberOfItemsView.scala.html

```diff
@@ -0,0 +1,48 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.components.{Button, ErrorSummary, InputText, HeadingCaption}
+@import models.MovementReferenceNumber
+
+@this(
+    mainTemplate: MainTemplate,
+    formHelper: FormWithCSRF,
+    button: Button,
+    inputText: InputText,
+    errorSummary: ErrorSummary,
+    caption: HeadingCaption
```

Comment:
This and the subsequent import don't look to be being used

Reply:
removed